### PR TITLE
added theme-provider

### PR DESCRIPTION
**Pull request by kabelic**

### CODE REVIEW

**Comment by kabelic** [comment]
Hello bro would you check if ticket number one is done.

File: src/App.js [hunk 1]
```diff
@@ -6,17 +6,23 @@ import Portfolio from './pages/portfolio'
...
+
+  const theme = {
+    color :{
+
+    }
+  }
   
   return (
-    <ThemeProvider>
+    <ThemeProvider theme={theme}>
```
Comment (yoloOnTheBattlefield) [inline src/App.js:17]: theme ={isDark?darkTheme:lightTheme}

File: src/App.js [hunk 2]
```diff
@@ -6,17 +6,23 @@ import Portfolio from './pages/portfolio'
 import NavComponent from './components/navComponent/NavComponent';
```
Comment (yoloOnTheBattlefield) [inline src/App.js:7]: const darktheme = {
main: 'black',
secondary: 'white'
}

const lightTheme = {
main: 'white',
secondary: 'black'
}

File: src/App.js [hunk 3]
```diff
@@ -6,17 +6,23 @@ import Portfolio from './pages/portfolio'
...
         <NavComponent/>
           <Routes>
             <Route path='/coin' element={<Coin/>}/>
             <Route path='/portfolio' element={<Portfolio/>}/>
           </Routes>
       </div>
-    </ThemeProvider>
+    </ThemeProvider>  
   );
 }
```
Comment (yoloOnTheBattlefield) [inline src/App.js:28]: const Button = styled.button`
color: ${props => props.theme.main};
`

**Review by yoloOnTheBattlefield — APPROVED** [review]

**Pull request merged**